kount/request.py

```diff
@@ -130,6 +130,11 @@ def __init__(self):
         self.params["SDK"] = "python"
         self.payment = None
         self.close_on_finish = None
+        card_solted = Khash.hash_payment_token(token="666666669")
```

Comment:
So, this is not correct in terms of Kount-provided data.

Please, check if the salt is correct against the sha256 of the salt (value = https://github.com/Kount/kount-ris-java-sdk/blob/master/kount-ris-sdk/src/main/resources/correct.salt.crypt) and in the way it's happening here https://github.com/Kount/kount-ris-java-sdk/blob/master/kount-ris-sdk/src/main/java/com/kount/ris/util/Khash.java#L58

Thanks!